boule/_triaxialellipsoid.py

```diff
@@ -164,9 +166,44 @@ def _check_geocentric_grav_const(self, geocentric_grav_const, value):
 
     @property
     def mean_radius(self):
+        """
+        The mean radius of the ellipsoid. This is equivalent to the degree 0
+        spherical harmonic coefficient of the ellipsoid shape.
+        Definition: :math:`R_0`.
```

Comment:
Same here. Could be the same equation (without the longitude integration).

Reply:
updated as well.